src/game.cpp

```diff
@@ -2676,7 +2675,10 @@ bool game::load( const save_t &name )
     u = avatar();
     u.set_save_id( name.decoded_name() );
     const std::string save_filename = playerpath + SAVE_EXTENSION;
-    if( !read_from_file( save_filename, std::bind( &game::unserialize, this, _1, save_filename ) ) ) {
+    if( !read_from_file( save_filename,
+    [&]( std::istream & is ) {
```

Comment:
AFAIK we avoid capture-defaults because it isn't clear what exactly is captured and we might reuse variables accidentally.